exercises/01_trim_right/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func InSet(b byte, set []byte) bool {
+	for _, v := range set {
+		if v == b {
+			return true
+		}
+	}
+	return false
+}
+
+var trimSet = []byte(" \r\n\t")
+
+func TrimRight(s string) string {
+	for s != "" && InSet(s[len(s)-1], trimSet) {
+		s = s[:len(s)-1]
+	}
+	return s
+}
+
+func main() {
+	fmt.Print("Enter a string: ")
```

Comment:
nit: Please drop this confusing string. Input string `Enter a string: Enter text: ` looks weird